### ISSUE TITLE
Runtime Error after building and installing pytorch wrapper

### ISSUE THREAD

**Issue by ZaidQureshi** [NVIDIA/nvvl#42]
When I try to run the benchmark.py script after building and installing the pytorch wrapper with:
```
python setup.py install --with-nvvl=../build
```
python compalins that libnvvl.so is not found:
```
ImportError: libnvvl.so: cannot open shared object file: No such file or directory
```

When I add the libnvvl path to LD_LIBRARY_PATH like:
```
export LD_LIBRARY_PATH=/workspace/build/:$LD_LIBRARY_PATH 
```
I run into a runtime error with the benchmark.py script saying:
```
terminate called after throwing an instance of 'std::runtime_error'
  what():  0: I give up, I can't get it to seek to frame 55
Aborted (core dumped)
```

I have tried compiling in the docker container and outside but nothing seems to work. I need to modify the src/VideoLoader.cpp file for some profiling. If I don't recompile the pytorch wrapper, my changes aren't visible when I run the benchmark.py script.

**Comment by jaredcasper** [comment]
What is the container format and codec of the video you are using?  That error means that for whatever reason libavformat isn't able to seek to frame 55, even when asking libavformat to seek to frame 0, it still returns a frame later than frame 55. This can happen if you are using h265 with large open GOPs (see the README for how to encode with smaller closed gops), or maybe if you are using a container that doesn't support seeking very well. The Matroska (MKV) container has the best support for seeking, so you could try to remux it into an .mkv.

**Comment by ZaidQureshi** [comment]
I have prepared the data according to the directions and scripts provided here:
https://github.com/NVIDIA/nvvl/tree/master/examples/pytorch_superres
Also, this doesn't appear as a problem when I am just using the provided code without recompiling the python wrapper.

**Comment by ZaidQureshi** [comment]
So I think its actually a problem with the modifications I have made to the VideoLoader.cpp file to profile the reading of frames.
My modified file can be found at:
https://gist.github.com/ZaidQureshi/17ea2598239b67c5bf592c2f2aa24561
I have modified the read_file function and taken out the av_read_frame call out of the loop condition to profile it. So I have had to add the call to it once before the loop, once before each continue (Lines 400 and 438) and once at the end of the loop at Line 531. I am not sure what mistake I have made.
